image_tools_py/src/showimage_py.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# System imports
+import sys
+import argparse
+
+# ROS2 imports
+import rclpy
+import sensor_msgs.msg
+
+# OpenCV imports
+import cv2
+
+import numpy
+
+# Local imports
+import image_qos_profile
+
+args = None
```

Comment:
Instead of making `args` global I would recommend to create a lambda-style / locally defined callback function. So the callback function can use the `args` from the outer scope but the `args` don't have to be global.